client/src/components/PacmanGame/PacmanGame.jsx

```diff
@@ -1,59 +1,47 @@
 import React, { Component } from 'react';
-// import Konva from 'konva';
-// import { Stage, Layer, Text, Circle, Line, Rect } from 'react-konva';
+import PropTypes from 'prop-types';
 
-import { initGridState, getFoods, getGhosts, getPacman, getWalls, getEnergizers, entityToCode } from './gameCore';
+import {
+  initGridState, getFoods, getGhosts,
+  getPacman, getWalls, getEnergizers,
+  entityToCode, entityApplier,
+} from './gameCore';
 
-import { PacmanBoard } from './PacmanBoard';
+import PacmanBoard from './PacmanBoard';
 
 class PacmanGame extends Component {
   constructor(props) {
-    super(props)
+    super(props);
     this.state = {
       gridState: '',
     };
   }
 
   componentDidMount() {
-    const canvasWidth = this.props.width;
-    const cellsInEachRow = this.props.numberofCells;
+    const { width, numberofCells } = this.props;
+    const canvasWidth = width;
+    const cellsInEachRow = numberofCells;
     const gridSize = canvasWidth / cellsInEachRow;
-    let gridState = initGridState(cellsInEachRow, cellsInEachRow);
+    const gridState = initGridState(cellsInEachRow, cellsInEachRow);
```

Comment:
Instead of passing multiple arguments to a function, try to create a single argument as object
```suggestion
    const gridState = initGridState({ a: cellsInEachRow, b: cellsInEachRow });
```
Pros: You know the what are args before looking at function argument, you don't have to remember arguments sequence.

Reply:
Will use in future multiple arguments will need to be passed. 
For this function, I converted it to `initSquareGridState(edge)`